[iOS] Undefined symbols with latest GraalVM

<!--- Provide a brief summary of the issue -->
Integration tests on iOS fail with GraalVM from head:
```
[Fri Mar 25 03:06:13 CET 2022][INFO] [SUB] Undefined symbols for architecture arm64:
[Fri Mar 25 03:06:13 CET 2022][INFO] [SUB]   "_systemVersionPlatform", referenced from:
[Fri Mar 25 03:06:13 CET 2022][INFO] [SUB]       ___svm_vm_is_static_binary in com.gluonhq.substrate.test.main.o
[Fri Mar 25 03:06:13 CET 2022][INFO] [SUB]   "_systemVersionPlatformFallback", referenced from:
[Fri Mar 25 03:06:13 CET 2022][INFO] [SUB]       ___svm_vm_is_static_binary in com.gluonhq.substrate.test.main.o
```

### Expected Behavior
<!--- Tell us what should happen -->

### Current Behavior
<!--- Tell us what happens instead of the expected behavior -->

### Steps to Reproduce
<!--- Steps to reproduce this bug -->

### Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->

--- a/src/main/java/com/gluonhq/substrate/Constants.java
+++ b/src/main/java/com/gluonhq/substrate/Constants.java
@@ -110,7 +110,7 @@ public class Constants {
     public static final String DEFAULT_JAVAFX_STATIC_SDK_VERSION  = "18-ea+11";
     public static final String DEFAULT_JAVAFX_JS_SDK_VERSION  = "18-internal+0-2021-09-02-165800";
     public static final String DEFAULT_SYSROOT_VERSION  = "20210424";
-    public static final String DEFAULT_CLIBS_VERSION  = "26";
+    public static final String DEFAULT_CLIBS_VERSION  = "27";
     public static final String DEFAULT_JAVASDK_PATH = "staticjdk";
     public static final String DEFAULT_JAVASDK_PATH11 = "labs-staticjdk";
 

--- a/src/main/java/com/gluonhq/substrate/Constants.java
+++ b/src/main/java/com/gluonhq/substrate/Constants.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, 2021, Gluon
+ * Copyright (c) 2019, 2022, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by

--- a/src/main/resources/native/ios/JvmFuncsFallbacks.c
+++ b/src/main/resources/native/ios/JvmFuncsFallbacks.c
@@ -508,11 +508,6 @@ JNIEXPORT jobject JNICALL JVM_GetSystemPackages(JNIEnv *env) {
     return NULL;
 }
 
-JNIEXPORT jobject JNICALL JVM_GetTemporaryDirectory(JNIEnv *env) {
-    (*env)->FatalError(env, "JVM_GetTemporaryDirectory called:  Unimplemented");
-    return NULL;
-}
-
 JNIEXPORT jobject JNICALL JVM_GetVmArguments(JNIEnv *env) {
     (*env)->FatalError(env, "JVM_GetVmArguments called:  Unimplemented");
     return NULL;

